### DISCUSSION TITLE
Performing active learning on magic data

### DISCUSSION THREAD

**Discussion by nikokks** [mindee/doctr#1242] [Ideas]
### 🚀 The feature

I would like to know if you have already applied techniques to clean your magic data? 

Indeed, it only takes a few corrections of labeling errors on the data to achieve better performance:  like the active learning technique for example. 

After obtaining the checkpoint on the best val_loss, making an inference on the training data and ordering the data in descending order of loss would make it possible to see the few hundred (sometimes more) of mistagged data. 

With your agreement, I would be very interested in doing this task once the parseq model is trained :)

### Motivation, pitch

allows to boost models performances

**Comment by felixdittrich92** [comment]
Hey @nikokks If i understood it correctly .. i think to get access to the internal data isn't possible because it contains sensible personal data. Ping @odulcy-mindee @charlesmindee for a more data specific answer ^^
